python/3d/auto_generated/materials/qgsphongmaterialsettings.sip.in

```diff
@@ -60,12 +60,12 @@ Returns diffuse color component
 %Docstring
 Returns specular color component
 %End
-    double shininess() const;
+    float shininess() const;
```

Comment:
This was deliberate -- it was to avoid the user-set values changing for them when saving/restoring projects. We can safely convert to float when building the material, but doubles should be used for storing the value to avoid this behaviour.

Reply:
> I guess that the textured case should use double

Yes please!

Reply:
Done.